app/scripts/modules/azure/loadBalancer/configure/listeners.html

```diff
@@ -14,7 +14,7 @@
         <tbody>
           <tr ng-repeat="rule in loadBalancer.loadBalancingRules">
             <td>
-              <select class="form-control input-sm" ng-model="rule.protocol" ng-options="protocol for protocol in ['UDP','TCP']"></select>
+              <select class="form-control input-sm" ng-model="rule.protocol" ng-options="protocol for protocol in ['HTTP']"></select>
```

Comment:
Are additional protocols expected? If not, it's probably worth removing this input.